packages/apple-pay-integration/src/apple-pay-customer-strategy.ts

```diff
@@ -81,6 +82,15 @@ export default class ApplePayCustomerStrategy implements CustomerStrategy {
         this._onClick = onClick;
 
         let state = this._paymentIntegrationService.getState();
+        const { features } = state.getStoreConfigOrThrow().checkoutSettings;
+
+        this._isWebBrowserSupported = features['PAYPAL-4324.applepay_web_browser_support'];
```

Comment:
Can we assign a fallback back as `true`?

Reply:
I think this might be ok as it reads the value from experiment 🤔